packages/vite/src/node/optimizer/registerMissing.ts

```diff
@@ -47,14 +47,20 @@ export function createOptimizedDeps(server: ViteDevServer): OptimizedDeps {
 
   let newDepsToLog: string[] = []
   let newDepsToLogHandle: NodeJS.Timeout | undefined
-  const logNewDeps = () => {
-    config.logger.info(
-      colors.green(`✨ dependencies optimized: ${depsLogString(newDepsToLog)}`),
-      {
-        timestamp: true
-      }
-    )
-    newDepsToLog = []
+  const logNewlyDiscoveredDeps = () => {
+    if (newDepsToLog.length) {
+      config.logger.info(
+        colors.green(
+          `✨ newfound dependencies optimized: ${depsLogString(
```

Comment:
```suggestion
          `✨ new dependencies optimized: ${depsLogString(
```

Perhaps it could be "new" only, "newfound" looks a bit off 😅

Reply:
New found? They arent really new, they were just not discovered before

Reply:
Before we had "new dependencies found", so maybe that is an option, since we arent listing the optimized logs for the others

Reply:
"new dependencies optimized" sounds good 👍🏼